app/jobs/socure_shadow_mode_proofing_job.rb

```diff
@@ -91,6 +92,10 @@ def build_applicant(
     )
 
     applicant_pii = decrypted_arguments[:applicant_pii]
+    if applicant_pii[:phone].nil? && applicant_pii[:best_effort_phone_number_for_socure]
```

Comment:
Note that, in actuality, nothing is passing `:phone` into the Socure proofer today, so this conditional isn't strictly necessary. But it feels cleaner, and IMHO makes clearer what we're doing. (Plus it frees me from needing nil-safe operators on `applicant_pii[:best_effort_phone_number_for_socure]` and keeps `phone` and `phone_source` together.)